src/bootstrapEditor.js

```diff
@@ -29,7 +29,7 @@ function InputNumber(props) {
     <FormGroup
       controlId={name}
       className="input20Percent"
-      inline
+      inline="true"
```

Comment:
I think we could remove this line entirely. what do you think? Did you see any improvements adding "=true"?

Reply:
I added a Boolean value because i saw this property as Boolean in the requirements of "react-bootstrap". Nevertheless, i still got it wrong because it inserted a string ("true"), instead of true.
Also my compilation had error because of the inline prop.

Reply:
Hum. Strange you got an error when compiling. React requires to have boolean props inserted without the "=" symbol. Anyway, I dont remember why "inline" was used for. And after small tests, there is no difference when removed. So remove the props entirely if that's a problem when compiling.